src/hipscat/io/paths.py

```diff
@@ -104,24 +110,57 @@ def pixel_catalog_files(
         catalog_base_dir (FilePointer): base directory of the catalog (includes catalog name)
         pixels (List[HealpixPixel]): the healpix pixels to create pointers to
         storage_options (dict): the storage options for the file system to target when generating the paths
+        query_params (dict): query parameters to append to the url for http file systems.
+            An example of the expected `query_params` dictionary format:
+            {
```

Comment:
This block is rendered oddly in the API docs:

https://hipscat--291.org.readthedocs.build/en/291/autoapi/hipscat/io/paths/index.html#hipscat.io.paths.pixel_catalog_files